Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Reworked elemental doctrines

## Changelog:
- Reworked spirit and decay magical doctrines for future usage:
  -  It's needed for future branches, so characters can potentially bully water elementals while having a good relationship with fire elementals, etc.
  - Removed 2 old (spirit&decay) doctrines
  - Reworked 2 and added 3 new sub-doctrines for each of 4 elements doctrines
 - Changed a few starting doctrines for various faiths

## Dev changelog:
- Reworked a few triggers, replaced several triggers with one.

## Tests:

# How to test:
- Make sure that tooltips correctly inform you why this specific doctrine can not be picked when your character reforms or create the new faith.
- Tell me if any of the elemental doctrines are assigned wrongly for a faith. Some clarifications:
  - Controlled  means that mages of that faith are allowed to use this element
  - Harmonised means that this element is allowed to be communed with by shamans of that faith, and Idolised-Spirit means that the faith promotes shamans to commune with this element
  - Enslaved means that this element is allowed to be enslaved by dark shamans of that faith, and Idolised-Decay means that the faith promotes dark shamans to enslave this element
